test/criteria.jl

```diff
@@ -150,4 +151,12 @@ end
     @test stopping_time(Patience(n=1), losses) == 3
 end
 
+@testset "MaximumChecks" begin
+    @test_throws ArgumentError MaximumChecks(n=0)
+
+    for i in 1:6
+        @test stopping_time(MaximumChecks(i), losses) == i+1
```

Comment:
Why is the implementation requirement not this:

```julia
 @test stopping_time(MaximumChecks(i), losses) == i
```

?

Reply:
Well, depends on the concept  of the limit, and how done is going to be used, maybe I was wrong about when done is going to be used. I have updated the [commit](https://github.com/ablaom/EarlyStopping.jl/pull/3/commits/bbb6767923337644ac250a69265adaaf9ea2f324).